packages/kit/test/apps/basics/src/routes/redirect/crashing/__error.svelte

```diff
@@ -0,0 +1,11 @@
+<script context="module">
+	/** @type {import('@sveltejs/kit').ErrorLoad} */
+	export async function load() {
+		return {
+			status: 307,
+			redirect: './c'
```

Comment:
This line suggests a redirection to `redirect/crashing/c`, but it actually redirects to `redirect/c`.

It might creation confusion.

Reply:
Yeah, I'd have expected it to redirect to `/redirect/crashing/c`. Is it expected that it doesn't or does it seem like a bug? Maybe this is an instance of https://github.com/sveltejs/kit/issues/2140

Reply:
It seems to be an instance of #2140 indeed.

The suggested fix of setting `trailingSlash` to `'always'` fixes it for the `crashing/__error` redirection.
But then `redirect` tests are now failing, since `redirect/a` tries to redirect to `redirect/a/b` instead of `redirect/a`. This seems to be the case for any non-`__error` redirections.

My concern is then that relative redirection behaviour is not consistent between an `__error` redirection and a regular page redirection.

I find it weird, because i don't get why the redirection path would be resolved differently if originated from an `__error` page...

Reply:
Maybe it'd be best to redirect to `../c` in `redirect/crashing/__error`, and let this test fail (and PR open) until the inconsistency is fixed.

Reply:
I need to investigate more, but i think that redirection path resolution is broken when using relative URLs only on `__layout`, `__error`, and `index` pages.

Reply:
See #2235

Reply:
Ok, following discussion in #2235, the `redirect` of `crashing/__error` should be `./c`.

Tests should all pass now.